wgpu-core/src/command/bundle.rs

```diff
@@ -746,7 +746,21 @@ pub struct RenderBundle<A: HalApi> {
     pub(crate) life_guard: LifeGuard,
 }
 
+#[cfg(any(
```

Comment:
Do we actually need to do this for wgpu-core and hal? Or would forcing things at the `wgpu` layer work?

Reply:
Maybe? I assume it was implemented initially here for a reason.
I can try and play around with that idea.

Reply:
Well, after scrolling through some of the places where this is done, there might be some types where this isn't necessary, but there are plenty of other implementations that require this, like callback types.

So I would say maybe, but we couldn't remove the crate feature from `core` or `hal` completely anyway.

Basically what I did in this PR is just gate the already existing `Send` and `Sync` implementations, I assumed they were there for a good reason. If they aren't then I would argue we can figure this out in a different PR, it takes me quite a bit of time to understand the internals enough to figure out if this was unnecessary or not.

Reply:
Alright, let's keep it as is then, was just trying to reduce conflicts with arcinization.